### PR TITLE
docs: fr translation

### PR DESCRIPTION
**Pull request by clemcode**

### CODE REVIEW

File: content/fr/0.docs/1.get-started/4.commands.md [hunk 1]
```diff
@@ -173,13 +173,13 @@ Depuis Nuxt v2.13, un robot d'exploration est installé qui va maintenant explor
 
 <base-alert>
```
Comment (Flosciante) [inline content/fr/0.docs/1.get-started/4.commands.md]: wrong syntax

File: content/fr/0.docs/1.get-started/4.commands.md [hunk 2]
```diff
@@ -173,13 +173,13 @@ Depuis Nuxt v2.13, un robot d'exploration est installé qui va maintenant explor
...
 
-**Attention:** les routes dynamiques sont ignorées par la commande `generate` lors de l'utilisation Nuxt <= v2.12: [API de configuration de la génération](/docs/2.x/configuration-glossary/configuration-generate)
+**Attention:** les routes dynamiques sont ignorées par la commande `generate` lors de l'utilisation Nuxt <= v2.12: [API de configuration de la génération](/docs/configuration-glossary/configuration-generate)
 
 </base-alert>
 
 <base-alert type="info">
```
Comment (Flosciante) [inline content/fr/0.docs/1.get-started/4.commands.md]: wrong syntax

File: content/fr/0.docs/1.get-started/4.commands.md [hunk 3]
```diff
@@ -209,7 +209,7 @@ npm run generate --fail-on-error
 
 <base-alert type="next">
```
Comment (Flosciante) [inline content/fr/0.docs/1.get-started/4.commands.md]: wrong syntax

File: content/fr/2.blog/6.going-full-static.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+To improve the user experience as well as telling Nuxt that you want to export your application to static hosting, we are introducing a `target` option in your `nuxt.config.js`:
+
+```js{}[nuxt.config.js]
+export default {
+  target: 'static' // default is 'server'
+}
+```
+
+<base-alert type="warning">
```
Comment (Flosciante) [inline content/fr/2.blog/6.going-full-static.md:92]: wrong syntax

File: content/fr/0.docs/2.concepts/1.views.md [hunk 5]
```diff
@@ -92,11 +63,11 @@ Nous pouvons définir un layout par défaut en ajoutant un fichier `default.vue`
 </template>
 ```
 
-<base-alert type="next">
+::alert{type="next"}
 
 Se référer au [composant Nuxt](/docs/2.x/features/nuxt-components) dans le chapitre des composants.
```
Comment (Flosciante) [inline content/fr/0.docs/2.concepts/1.views.md]: To remove 2.x and add /fr on link

File: content/fr/0.docs/3.features/5.meta-tags-seo.md [hunk 6]
```diff
@@ -215,14 +193,8 @@ Ajoutez des liens de `prefetch` et de `preload` pour accélérer le temps de cha
 
 Nous voudrons peut-être désactiver cette option si nous avons beaucoup de pages et de routes.
 
-<base-alert type="next">
+::alert{type="next"}
 
 [Conseils pour les ressources](/docs/2.x/configuration-glossary/configuration-render#resourcehints).
```
Comment (Flosciante) [inline content/fr/0.docs/3.features/5.meta-tags-seo.md]: To remove 2.x and add /fr on link

File: content/fr/0.docs/7.components-glossary/9.validate.md [hunk 7]
```diff
@@ -1,12 +1,14 @@
 ---
```
Comment (Flosciante) [inline content/fr/0.docs/7.components-glossary/9.validate.md:1]: To remove 2.x and add /fr on links in this file

**Review by Flosciante — CHANGES_REQUESTED** [review]

File: content/fr/0.docs/2.concepts/1.views.md (hunk 5 above)
Comment (clemcode) [reply content/fr/0.docs/2.concepts/1.views.md]: I think Docus i18n should handle the /fr part actually, as we can host translation under subdomains as well

**Review by Flosciante — APPROVED** [review]

**Pull request merged**